bitwarden-rs-alpine.json

```
@@ -1,16 +1,16 @@
 {
     "Bitwarden-rs": {
-        "description": "Bitwarden password manager server in Rust.",
+        "description": "Bitwarden password manager server in Rust and on Alpine (speed, size). Uses <a href=\"https://hub.docker.com/r/bitwardenrs/server\">bitwardenrs/server</a>.",
```

Harmonize style with other rock-ons:

```suggestion
        "description": "Bitwarden password manager server in Rust and on Alpine (speed, size). <p>Based on the following docker image: <a href='https://hub.docker.com/r/bitwardenrs/server' target='_blank'>https://hub.docker.com/r/bitwardenrs/server</a></p>.",
```